src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -673,6 +675,70 @@ def detect(
                 how='diagonal',
             )
 
+    def measure_samples(
+            self,
+            method: str | Callable[..., pl.Expr],
+            **kwargs: Any,
+    ) -> pl.DataFrame:
+        """Calculate eye movement measure for gaze data samples.
+
+        If :py:class:``GazeDataFrame`` has :py:attr:``trial_columns``, measures will be grouped by
+        trials.
+
+        Parameters
+        ----------
+        method: str | Callable[..., pl.Expr]
+            Measure to be calculated.
+        **kwargs: Any
+            Keyword arguments to be passed to the respective measure function.
+
+        Returns
+        -------
+        pl.DataFrame
+            Measure results.
+
+        Examples
+        --------
+        Let's initialize an example GazeDataFrame first:
+        >>> gaze = pm.gaze.from_numpy(
+        ...     distance=np.concatenate([np.zeros(40), np.full(10, np.nan), np.ones(50)]),
+        ... )
+
+        You can calculate measures, for example the null ratio like this:
+        >>> gaze.measure_samples('null_ratio', column='distance')
+        shape: (1, 1)
+        ┌────────────┐
+        │ null_ratio │
+        │ ---        │
+        │ f64        │
+        ╞════════════╡
+        │ 0.1        │
+        └────────────┘
+        """
+        if isinstance(method, str):
+            method = pm.measure.MeasureLibrary.get(method)
+
+        method_args = inspect.getfullargspec(method).args
+        if 'column_dtype' in method_args:
```

Comment:
does this fit in one line?

Reply:
you mean like this? 

```python
if 'column_dtype' in method_args: kwargs['column_dtype'] = self.frame[kwargs['column']].dtype
```

including the leading spaces this has 101 characters, but I also consider that less readable than two lines